conf/modules.config

```diff
@@ -691,6 +691,7 @@ process {
     // CONTAMINATION ESTIMATION
     //
     withName: ANGSD_DOCOUNTS {
+        ext.prefix = { "${meta.id}_${meta.library_id}_${meta.reference}" }
```

Comment:
Keep the order decided for module config attributes in `docs/development/code_conventions.md`